x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.tsx

```diff
@@ -268,7 +268,7 @@ export const CollapsedItemActions: React.FunctionComponent<ComponentOpts> = ({
           'data-test-subj': 'deleteRule',
           onClick: () => {
             setIsPopoverOpen(!isPopoverOpen);
-            setRulesToDelete([item.id]);
+            onDeleteRule(item);
```

Comment:
I think you do not need to pass `item` as a param here?
I can see in rule list that `onDeleteRule` looks like:
`
onDeleteRule={() =>
                    updateRulesToBulkEdit({
                      action: 'delete',
                      rules: [rule],
                    })
                  }
`

Reply:
I think we should keep it just to have a more consistent API since all of the other `onSomethingRule` handlers pass it in

Reply:
On my opinion, it's misleading.